src/test/java/berlin/yuna/nano/services/metric/logic/MetricServiceTest.java

```diff
@@ -0,0 +1,93 @@
+package berlin.yuna.nano.services.metric.logic;
+
+import berlin.yuna.nano.core.Nano;
+import berlin.yuna.nano.helper.logger.logic.LogQueue;
+import berlin.yuna.nano.helper.logger.model.LogLevel;
+import berlin.yuna.nano.services.http.HttpService;
+import berlin.yuna.nano.services.http.model.HttpMethod;
+import berlin.yuna.nano.services.http.model.HttpObject;
+import org.junit.jupiter.api.Test;
+
+import java.util.Map;
+
+import static berlin.yuna.nano.core.model.Config.*;
+import static org.assertj.core.api.Assertions.assertThat;
+
+class MetricServiceTest {
+
+    protected static String serverUrl = "http://localhost:";
+
+    @Test
+    void metricEndpointsWithoutBasePath() {
+        Nano nano = new Nano(Map.of(CONFIG_LOG_LEVEL, LogLevel.INFO, CONFIG_LOG_FORMATTER, "console"), new LogQueue(), new MetricService(), new HttpService());
```

Comment:
Use global test LogLevel: `Nano(Map.of(CONFIG_LOG_LEVEL, TEST_LOG_LEVEL)`. Makes it easier to switch log levels while debugging